lib/common/xml.c

```diff
@@ -559,6 +574,15 @@ fix_plus_plus_recursive(xmlNode * target)
     }
 }
 
+/*!
+ * \brief If the attribute value is not already set, then set it
```

Comment:
The important thing this function does is replace XML attribute values like value="value++" with the previous value plus 1, or something like value="value+=5" to the previous value plus 5.

It's an undocumented feature, but users are allowed to do this in the CIB for any integer values. I suspect it would only ever be useful when setting node attributes via the CIB.